Makefile

```diff
@@ -101,6 +101,12 @@ lint-js:
 	npm run lint
 	@echo ""
 
+test-build-js:
+	@echo "--> Running JavaScript build"
+	npm install gulp
+	./node_modules/.bin/gulp build && node_modules/.bin/gulp package
```

Comment:
`make test` covers `./node_modules/.bin/gulp build && node_modules/.bin/gulp package`, so this might not be needed. Also `make` covers these 2 things if needed to run locally.

Reply:
Thank you,@charhate, for comments. In my experience tests has been passed with very old node js, but not `./node_modules/.bin/gulp build && node_modules/.bin/gulp package`. Therefore I added new command to make sure that everything is okay.  I'm not frontend programmer, I think. it might be happened because test coverage,

P.S.
But if you prefer, I can rollback this changes, of course.

Reply:
I'v remove that changes, I will try to add javascript tests to Travis.

Reply:
I'v update first post. I'd added `make test-js`